openmc/executor.py

```diff
@@ -0,0 +1,212 @@
+from collections.abc import Iterable
+import subprocess
+from numbers import Integral
+
+import openmc
+from openmc import VolumeCalculation
```

Comment:
This import is unused. VolumeCalculation is only used in comments, and that too in its whole form i.e. `openmc. VolumeCalculation`